platform/email/platform-email-impl/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Email Service"
+         id="org.codice.ddf.platform.email.impl.EmailServiceImpl"
+         description="Configure the hostname (or IP address) and port number of an SMTP mail server that will receive the emails.">
+
+        <AD
+                description="Mail server hostname (must be resolvable by DNS) or IP address."
+                name="Host" id="hostName" required="true"
+                type="String" default=""/>
+
+        <AD
+                description="Mail server port number."
+                name="Port" id="portNumber" required="false"
+                type="Integer" default="25"/>
+
+        <AD
+                description="Mail server user name."
+                name="User Name" id="userName" required="false"
+                type="String" default=""/>
+
+        <AD
+                description="Mail server password."
+                name="Password" id="password" required="false"
+                type="String" default=""/>
```

Comment:
@glenhein You can find an example of how to do it here: https://github.com/codice/ddf/blob/master/platform/security/sts/security-sts-ldaplogin/src/main/resources/OSGI-INF/metatype/metatype.xml